config/councils.yml

```diff
@@ -22,6 +26,8 @@ default: &default
     logo_path: "logo-camden.svg"
     privacy_link: "https://www.camden.gov.uk/data-protection-privacy-and-cookies"
     default_from_address: "help@camden.beacon.support"
+    invite_template: '31d2b530-8db4-44ce-9bfa-34b01519335f'
```

Comment:
Are we just using Camden's template for everything at the moment? If so, we should probably just use the Beacon logo instead of the Council's logo so that we can demo something neutral.

Reply:
we can't use beacon's logo because the logo is tied to the notify account we're using, which is in turn tied to the email address of whoever creates the account (camden emma in this case)